modules/compatibility/src/test/java/org/apache/ignite/compatibility/testframework/util/MavenUtils.java

```diff
@@ -148,7 +151,8 @@ private static void downloadArtifact(String artifact) throws Exception {
         X.println("Downloading artifact... Identifier: " + artifact);
 
         exec(buildMvnCommand() + " org.apache.maven.plugins:maven-dependency-plugin:3.0.2:get -Dartifact=" + artifact +
-            (useGgRepo ? " -DremoteRepositories=" + GG_MVN_REPO : ""));
+            (useGgRepo ? " -DremoteRepositories=" + GG_MVN_REPO : "") +
```

Comment:
Not sure what will happen with both `-DremoteRepositories` and `-s <path to settings.xml>` defined, but would recommend to use `-DremoteRepositories` only if `<path to settings.xml>` is unavailable.

Reply:
Agreed. Please take a look at the latest version.